bin/agat_sp_manage_functional_annotation.pl

```diff
@@ -1055,7 +1191,7 @@ =head1 DESCRIPTION
 
 =head1 SYNOPSIS
 
-    agat_sp_manage_functional_annotation.pl -f infile.gff [ -b blast_infile --db uniprot.fasta -i interpro_infile.tsv --id ABCDEF --output outfile ]
+    agat_sp_manage_functional_annotation.pl -f infile.gff [-b blast_infile][-d uniprot.fasta][-i interpro_infile.tsv][-id ABCDEF][-a][-o output]
```

Comment:
double dash for id, no?

Reply:
No, I don't think so.This is an issue I noticed but didn't come around to test: You have several options starting with the same letter. Default behavior for Getopt::Long is to warn if short style (`-d`) are used, but be ok if a unambiguos long style is used. Personally, I try to name the options differently to avoid collisions